demos/python_demos/llm_text_generation/README.md

```diff
@@ -1,14 +1,27 @@
-# Llama demo with python node {#ovms_demo_python_llama}
+# LLM text generation with python node {#ovms_demo_python_llm_text_generation}
+
+This demo shows how to take advantage of OpenVINO Model Server to generate content remotely with LLM models. 
+The server is running here the user provided python code in a MediaPipe Python calculator. We manage the generation with Hugging Faces pipeline and OpenVINO Runtime as the execution engine.
```

Comment:
The first sentence is quite ambiguous to me. For the second one, maybe we can use simpler language?
Like: "We use Hugging Face Optimum with OpenVINO Runtime as execution engine." ?
And for the future it's HuggingFace not Faces 😄

Reply:
done

Reply:
done